### PR TITLE
feat: ScrollArea (1/3)

### PR DESCRIPTION
**Pull request by zernonia**

### CODE REVIEW

**Comment by zernonia** [comment]
fianlly some progress!! this component is more complicated than I thought! 

https://github.com/radix-vue/radix-vue/assets/59365435/72fd4c3d-bd73-41d4-9fb3-287d967e2987

**Review by khairulhaaziq — APPROVED** [review]
Lgtm!

File: packages/radix-vue/src/ScrollArea/ScrollAreaViewport.vue [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+       */
+      overflowX: injectedValueFromRoot?.scrollbarXEnabled.value
+        ? 'scroll'
+        : 'hidden',
+      overflowY: injectedValueFromRoot?.scrollbarYEnabled.value
+        ? 'scroll'
+        : 'hidden',
+    }"
+  >
+    <div ref="contentElement" :style="{ minWidth: '100%', display: 'table' }">
```
Comment (simonbrunel) [inline packages/radix-vue/src/ScrollArea/ScrollAreaViewport.vue:45]: @zernonia I'm experiencing some layout issues with this component and it seems caused by this `display: 'table'`. What the reason behind using "table" here?

Comment (zernonia) [reply packages/radix-vue/src/ScrollArea/ScrollAreaViewport.vue:45]: We're not 100% sure regarding the reason for this, but I've tested out multiple case and seems like we don't really need it

**Pull request merged**